### ISSUE TITLE
TRS push specification

### ISSUE THREAD

**Issue by berezovskyi** [oslc-op/oslc-specs#565]
We should consider specifying how TRS clients can subscribe to receiving TRS CEs from TRS Servers without polling.

In the SCOTT project, we have implemented [TRS push over MQTT](https://github.com/EricssonResearch/scott-eu/tree/master/lyo-services/lib-common/src/main/kotlin/eu/scott/warehouse/lib). @jamsden liked the consideration of the MQTT transport. @axelreichwein suggested to consider Kafka implementation instead.

There are two more transports worth considering: WebSockets and Server Sent Events. Mark Nottingham, an HTTP expert at IETF (he was the one approving our OSLC HTTP header applications), wrote a post on the two: https://www.mnot.net/blog/2022/02/20/websockets. He suggests that WebSocket is too much of a blank canvas to be implemented efficiently at scale (though the scale he is talking about is likely beyond the intranet deployment of OSLC at any scale) for pubsub scenarios (short of protocol extensions), SSE is good but would likely work well only over HTTP/2 (not an issue in 2022) or HTTP/3, and Mark shows a proposed extension to HTTP that would be even better than SSE.

In short, I think it would be good to specify the push semantics and necessary resource shape changes in a protocol-independent fashion and provide at least 2 push transports in the spec: 1-2 for the backend clients (MQTT and Kafka) and at least 1 for browser clients (SSE).

**Comment by berezovskyi** [comment]
For SSE, we can specify two message types, e.g. `trs-ce:application/trs+json` and `trs-ce:application/ld+json` for a predefined JSON payload and a JSON-LD serialization of the proper RDF CE resource.
